admin/spec/components/previews/solidus_admin/ui/forms/text_field/component_preview.rb

```diff
@@ -6,7 +6,11 @@ class SolidusAdmin::UI::Forms::TextField::ComponentPreview < ViewComponent::Prev
 
   # The text field component is used to render a text field in a form.
   #
-  # It must be used within the block context yielded in the [`form_with`
+  # Most commonly, it'll be used indirectly through the definition given to a
+  # [form component](../form/overview).
+  #
+  # For standalone usage, it must be used within the block context yielded in
+  # the [`form_with`
   # ](https://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormHelper.html#method-i-form_with)
```

Comment:
I think there is an extra line here

Reply:
Well, that was the automatic formatting to make the line fit the max length :slightly_smiling_face: